### PR TITLE
Feature/hatem python 1

### PR DESCRIPTION
**Pull request by Hatem29**
in this task:
I have created a list of dictionaries named books in which each dictionary represents the information of each book.  
Then, I have created a new list of the titels of books that are written by mohammed and available at the time.
Lastly, I have printed an approprite short description for each book, focusing on its main tag if recognized, utilizing the match statment in python.

### CODE REVIEW

File: FIRST-NAME_LAST-NAME.py
Comment (FaresAbuIram) [inline FIRST-NAME_LAST-NAME.py:1]: Please don't remove this file

File: Jibreen_Jboor.py
Comment (FaresAbuIram) [inline Jibreen_Jboor.py:1]: Please don't remove this file

File: Mohammad_Ballal.py
Comment (FaresAbuIram) [inline Mohammad_Ballal.py:1]: Please don't remove this file

**Review by FaresAbuIram — CHANGES_REQUESTED** [review]
Could you please update the description? Let’s make it more meaningful.

For example:

"In this task, we accomplished 1, 2, and 3."

File: FIRST-NAME_LAST-NAME.py
Comment (Hatem29) [reply FIRST-NAME_LAST-NAME.py:1]: by the files you mean the files that I have deleted or modified from the main?

Comment (FaresAbuIram) [reply FIRST-NAME_LAST-NAME.py:1]: If you check the main branch, you'll see it has some existing files. We don't want to remove any of them; just create a new file for your work.

**Review by FaresAbuIram — APPROVED** [review]
LGTM, nice work 


Please merge it

**Pull request merged**